test/unittest/backends/mongodb/__init__.py

```diff
@@ -0,0 +1,8 @@
+import os
+
+import pytest
+
+skip = not os.environ.get('SUPERDUPER_CONFIG', "").endswith('mongodb.yaml')
+
+if skip:
+    pytest.skip("Skipping this file for now", allow_module_level=True)
```

Comment:
We should move the whole ibis test module to the mongodb plugins next.

Reply:
Agreed.